src/appleseed.python/module.cpp

```diff
@@ -35,6 +35,7 @@
 
 // Forward declarations.
 void bind_aov();
+void bind_appleseed_info();
```

Comment:
Maybe that should be called `bind_appleseed()` since it's exposing the `Appleseed` class (more or less) to Python...